src/kirby/components/avatar/avatar.component.scss

```diff
@@ -6,6 +6,7 @@ $image-diameter-medium: 56px;
 $image-diameter-large:  96px;
 
 :host {
+  --kirby-badge-elevation: #{get-elevation(2)};
```

Comment:
Consider outputting elevations as css variables and having `get-elevation(2)` return `var(--kirby-elevation-z2)` etc to avoid repetition